src/linked_list.py

```diff
@@ -0,0 +1,99 @@
+"""Create a class for linked lists and a few methods to manipulate it."""
+
+
+class Node(object):
+    """Create class to represent individual elements in data structure."""
+
+    def __init__(self, data, next=None):
+        """Something."""
+        self.data = data
+        self.next = next
+
+
+class LinkedList(object):
+    """Create linked lists."""
+
+    # def __init__(self, head=None):
+    #     """Instantiate objects with a head defaulted to None."""
+    #     if isinstance(head, list):
+    #         node = Node(head[0])
+    #         self.head = node
+    #         for i in range(1, len(head)):
+    #             self.push(head[i])
+    #     else:
+    #         self.head = head
+
+    def __init__(self, iterable=None):
+        """Instantiate objects wi."""
+        self.head = None
+        if iterable and hasattr(iterable, "__iter__"):
```

Comment:
You can actually simplify the next few lines quite a bit. If you `try` to iterate over something which is not an iterable, Python will raise an error. So, just handle that error!